src/main/java/core/basesyntax/handler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.db.FruitShopStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
+    private FruitShopStorage fruitShopStorage;
+
+    @Override
+    public void handle(FruitTransaction fruitTransaction) {
+        fruitShopStorage.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
```

Comment:
You should use controler to initiallize FruitShopStorage otherwise you get NullPoinerException,
fix everywhere